container.go

```diff
@@ -43,6 +43,7 @@ type Container struct {
 	PidMode              string                        `yaml:"pid_mode" json:"pid_mode"`
 	ShmSize              int64                         `yaml:"shm_size" json:"shm_size"`
 	Labels               []string                      `yaml:"labels" json:"labels"`
+	StopTimeout          string                        `yaml:"stop_timeout" json:"stop_timeout"`
```

Comment:
you could use a UintString here
https://github.com/replicatedhq/libyaml/blob/e8bad50650727997f11bf01d21a8817a7974fbed/uint_string.go